vad_transcriber/wavTranscriber.py

```diff
@@ -58,10 +58,10 @@ def stt(ds, audio, fs):
 Retunns a tuple containing each of the model files (pb, scorer)
 '''
 def resolve_models(dirName):
-    pb = glob.glob(dirName + "/*.pb")[0]
+    pb = glob.glob(dirName + "/*.pbmm")[0]
     logging.debug("Found Model: %s" % pb)
 
-    scorer = glob.glob(dirName + "/kenlm.scorer")[0]
```

Comment:
Why this one? This is already the correct release file name

Reply:
To match the [docs](https://github.com/mozilla/DeepSpeech/releases/tag/v0.7.0) for release 0.7.0 that refer to "deepspeech-0.7.0-models.scorer"?

Reply:
Ah maybe, KenLM.scorer is the name under `data/lm`